pgmpy/independencies/Independencies.py

```diff
@@ -55,6 +55,9 @@ def __eq__(self, other):
         other_assertions = other.get_assertions()
         return all(self_independency in other_assertions for self_independency in self.get_assertions())
 
+    def __hash__(self):
+        return sum(hash(assertion) for assertion in self.get_assertions())
```

Comment:
@ankurankan Consider the case of two `Independencies` objects with same `assertions` but these `assertions` are misordered. The `str` will result in different hashes for both.